plutus-core/plutus-core/src/PlutusCore/Builtin/Runtime.hs

```diff
@@ -36,7 +36,7 @@ deriving stock instance Show (RuntimeScheme n)
 -- we use strictdata, so this is just for the purpose of completeness
 instance NFData (RuntimeScheme n) where
     rnf r = case r of
-        RuntimeSchemeResult    -> rwhnf r
+        RuntimeSchemeResult    -> ()
```

Comment:
`r` is forced by virtue of being matched on, so we don't need to force it once again.